mcs/class/Mono.Profiler.Log/Mono.Profiler.Log/LogProcessor.cs

```diff
@@ -414,17 +414,13 @@ LogEvent ReadEvent ()
 				}
 
 				case LogEventType.HeapRoots: {
-					// TODO: This entire event makes no sense.
 					var hre = new HeapRootsEvent ();
 					var list = new HeapRootsEvent.HeapRoot [(int) _reader.ReadULeb128 ()];
 
-					hre.MaxGenerationCollectionCount = (long) _reader.ReadULeb128 ();
-
 					for (var i = 0; i < list.Length; i++) {
 						list [i] = new HeapRootsEvent.HeapRoot {
-							ObjectPointer = ReadObject (),
-							Attributes = StreamHeader.FormatVersion == 13 ? (LogHeapRootAttributes) _reader.ReadByte () : (LogHeapRootAttributes) _reader.ReadULeb128 (),
-							ExtraInfo = (long) _reader.ReadULeb128 (),
+							AddressPointer = ReadPointer (),
+							ObjectPointer = ReadObject ()
```

Comment:
This is breaking support for v13, isn't it?

Reply:
This is something I missed from merging master, I will fix that.